src/app/public/modules/wait/wait.component.spec.ts

```diff
@@ -147,6 +150,77 @@ describe('Wait component', () => {
     expect(document.activeElement).toBe(document.body);
   }));
 
+  it(`should allow tab navigation and focus when a fullPage wait existed but now does not even if .
```

Comment:
I'm having trouble understanding this sentence. Better way to word it?

Reply:
I've reworded this; however, I still don't love the wording. This is a tough one to explain because of how nuanced this bug was. Let me know if you still don't like it :)